adapters/oidc/spring-boot/src/main/java/org/keycloak/adapters/springboot/KeycloakSpringBootProperties.java

```diff
@@ -23,7 +23,9 @@
 import java.util.ArrayList;
 import java.util.List;
 
-@ConfigurationProperties(prefix = "keycloak", ignoreUnknownFields = false)
+/* setting ignoreUnknownFields=true to avoid re-rebinding problem with property keycloak.config.resolver
+   when using spring cloud - see KEYCLOAK-2977 */
+@ConfigurationProperties(prefix = "keycloak", ignoreUnknownFields = true)
```

Comment:
It's true already, but I want to make it explicit that it should be true together with the comment.